conmon-rs/server/src/child_reaper.rs

```diff
@@ -151,12 +155,12 @@ impl ReapableChild {
                     }
                 }
             };
-            if tx.send(()).is_err() {
+            if killed_tx.send(()).is_err() {
                 bail!("the receiver dropped the watch")
             }
             Ok(())
         });
-        rx
+        (killed_rx, exit_rx)
```

Comment:
Can we rename the `killed` channel to something like `done`? Because it indicates that we actively killed a process which can cause confusion.